backtester/eventhandlers/strategies/rsi/rsi_test.go

```diff
@@ -50,37 +52,37 @@ func TestSetCustomSettings(t *testing.T) {
 
 	mappalopalous[rsiPeriodKey] = "14"
 	err = s.SetCustomSettings(mappalopalous)
-	if err != nil && err.Error() != "provided rsi-period value could not be parsed: 14" {
-		t.Error(err)
+	if !errors.Is(err, base.ErrInvalidCustomSettings) {
+		t.Errorf("expected: %v, reveived %v", base.ErrInvalidCustomSettings, err)
```

Comment:
reveived -> received (I before v except after v) 😆 It's happening across multiple files so a ctrl + shift + f is needed 🗡️

Reply:
More than 80 typos 🤙